src/NorthwindWebsite/API/Controllers/CategoriesController.cs

```diff
@@ -26,6 +30,12 @@ public async Task<ActionResult<List<Category>>> GetCategories()
         return categories.ToList();
     }
 
+    /// <summary>
+    /// Returns and image for a category by id.
+    /// </summary>
+    /// <param name="categoryId">Id of the category that contains the image.</param>
+    /// <response code="404">Unable to find a category with the mentioned id.</response>
```

Comment:
What about positive `response` code attribute?
This comment is relevant for the other controller actions as well